app/src/main/kotlin/io/intrepid/fablekotlin/screens/homescreen/HomescreenActivity.kt

```diff
@@ -14,6 +18,17 @@ class HomescreenActivity : BaseMvpActivity<HomescreenContract.Presenter>(), Home
     override fun createPresenter(configuration: PresenterConfiguration): HomescreenContract.Presenter =
             HomescreenPresenter(this, configuration)
 
+    override fun onViewCreated(savedInstanceState: Bundle?) {
+        super.onViewCreated(savedInstanceState)
+
+        if ((!getLoggedInStatus()) ||  SharedPreferencesManager.getInstance(this).token == null) run { startActivity(Intent(this, LoginActivity::class.java)) }
```

Comment:
Do you have the Intrepid Style guide setup on your AS? I dont think it allows { } on the same line but I could be wrong.

Reply:
I do, and it's selected, I'm having trouble figuring out if it's working, because something things I've been expecting to autoformat aren't going the way I expect and I couldn't tell if it was because of Kotlin or not.

Reply:
(after reimaging my computer I had to reinstall it)

Reply:
Hmm super weird, I'd ask Russel if theres anything special you need to do otherwise don't worry about it.